### ISSUE TITLE
Configure parameters for `rpmbuild`

### ISSUE THREAD

**Issue by nknapp** [mojohaus/rpm-maven-plugin#47]
Hello and thanks for your plugin. It has helped me a lot in the last months.

I have a minor problem though: Building larger RPMs take pretty long and I noticed that `Checking for unpackaged files` is responsible for that. I would like to try to disable this part by adding the `nocheck` argument to the `rpmbuild`-call, but there is no way to configure additional parameters in the POM.
I have already added the configuration
```
<defineStatements>
   <defineStatement>_unpackaged_files_terminate_build 0</defineStatement>
</defineStatements>
```
but this does not prevent `check-files' from being called.

Could you add an configuration option for CLI arguments that are appended to the `rpmbuild`-commandline?

**Comment by dantran** [comment]
@nknapp how my files do you have?  are they large?  what is the final size?

**Comment by dantran** [comment]
try this  http://stackoverflow.com/questions/9292243/rpmbuild-change-compression-format

**Comment by rickard-von-essen** [comment]
This is related to a fix for `brp-repack-jars`.

**Comment by nknapp** [comment]
@dantran The total size of the package is about 128 MB with default compression and about 600 MB with "%define _binary_payload w0.gzdio". The build time seems to be about the same for both alternatives. The package contains 182 files. The 8 largest files are 41M to 86M, 5 other files are larger than 1M, the rest is smaller.

I had put `_unpackaged_files_terminate_build 0` in the configuration because I had hoped that `rpmbuild` would not run `check-files` if the result didn't matter anyway, but I was wrong...

**Comment by nknapp** [comment]
I have found a omit `check-files` with a define-statement: 

```
<defineStatement>__check_files /bin/true</defineStatement>
```

This calls `bin/true` instead of calling the `check-files` script. It does reduce the build duration about 15 seconds for my build. I had hoped for more. I closing this issue, since it the statement achieves my goal.

If anybody is interested: I found out about this, using the hint from [this stack-overflow question](http://stackoverflow.com/questions/8076471/how-to-know-the-value-of-built-in-macro-in-rpm). I dumped the whole configuration with `rpm --showrc` and looked for `check-files` and there it was.

Thanks for your kind help
